readme.md

```diff
@@ -32,47 +32,17 @@ make unit-test
 
 ## Quick start
 
-The CLI augments  an [AnnData file](https://anndata.readthedocs.io/en/latest/) (\*.h5ad) with cellxgene schema required ontology terms using the logic defined in a yaml config file. This yaml file should indicate the values for the schema slots or mappings between the original values and the corresponding schema slots.
+The CLI validates an [AnnData file](https://anndata.readthedocs.io/en/latest/) (\*.h5ad) based on the cellxgene schema specifications.
 
-An example of a yaml config file looks like this:
+You can run the validation by doing the following:
 
 ```
-obs:
-  assay_ontology_term_id: EFO:0010550
-  ethnicity_ontology_term_id: unknown
-  sex: male
-  tissue_ontology_term_id: UBERON:0000970
-  cell_type_ontology_term_id:
-    sub_cluster_name:
-      Adrenocortical cells-1: CL:0002097
-      Photoreceptor cells-1: CL:0000210
-uns:
-  version:
-    corpora_schema_version: 1.1.0
-  organism: Homo sapiens
-  organism_ontology_term_id: NCBITaxon:9606
-  layer_descriptions:
-    X: log1p
-   raw.X: raw
-  title: Survey of human embryonic development
-fixup_gene_symbols:
-   X: log1p
-   raw.X: raw
+cellxgene-schema validate input.h5ad
 ```
 
-You can use the config file to augment a dataset with  the schema using:
+If the validation is succesful there will be a zero exit status, otherwise there will be error messages indicating why validation was unsuccesful  along with a non-zero exit status.
```

Comment:
I would cut this and simply use the schema guide. Otherwise, I'd update based on my editorial changes above to the schema guide.

I recommend spell checking your markdown - _succesful_ -> _successful_ - same for the _un_ versions.